demo/vue/src/components/VoiceWidget.vue

```diff
@@ -5,30 +5,22 @@
       <label>
         AccessKey obtained from
         <a href="https://console.picovoice.ai/">Picovoice Console</a>:
-        <input
-          type="text"
-          name="accessKey"
-          v-on:change="updateInputValue"
-          :disabled="isLoaded"
-        />
+        <input type="text" name="accessKey" v-on:change="updateInputValue" />
       </label>
-      <button class="start-button" v-on:click="initEngine" :disabled="isLoaded">
-          Start Porcupine
+      <button class="start-button" v-on:click="initEngine">
+        Start Porcupine
       </button>
     </h3>
     <h3>Loaded: {{ isLoaded }}</h3>
     <h3>Listening: {{ isListening }}</h3>
-    <h3>Error: {{ isError }}</h3>
-    <p class="error-message" v-if="isError">
-      {{ JSON.stringify(errorMessage) }}
+    <h3>Error: {{ error !== null }}</h3>
+    <p class="error-message" v-if="error !== null">
+      {{ error.toString() }}
     </p>
-    <button v-on:click="start" :disabled="!isLoaded || isError || isListening">
+    <button v-on:click="start" :disabled="!isLoaded || error || isListening">
       Start
     </button>
-    <button v-on:click="pause" :disabled="!isLoaded || isError || !isListening">
-      Pause
-    </button>
-    <button v-on:click="stop" :disabled="!isLoaded || isError || !isListening">
+    <button v-on:click="stop" :disabled="!isLoaded || error || !isListening">
```

Comment:
I've been including "release" here too to kinda reset the demo, not sure if that's something we want to do though? I don't have any strong opinions